protocol/git/README.md

```diff
@@ -0,0 +1,88 @@
+# Git Protocol
+
+A guide for programming within version control.
+
+Maintain a Repo
+---------------
+
+* Use [GitFlow] as a branch management strategy.
+* Avoid including files in source control that are specific to your
+  development machine or process.
+* Delete local and remote feature branches after merging.
+* Perform work in a feature branch.
+* Rebase frequently to incorporate upstream changes.
+* Use a [pull request] for code reviews.
+
+[GitFlow]: https://nvie.com/posts/a-successful-git-branching-model/
+[pull request]: https://help.github.com/articles/using-pull-requests/
+
+Write a Feature
```

Comment:
@SeanMcTex thoughts on including a graphic, like we do here? https://github.com/handsomecode/iOS-Guidelines/blob/master/VersionControl.md

Reply:
Holy moley, I hadn't discovered the iOS-Guidelines! Great stuff.

My thinking in not including a diagram was that our branching diagram would simply be showing how Gitflow works. Since we're not trying to explain Gitflow as a whole, adding a diagram seemed like an incomplete explanation. But if you think it would add more than it detracts, I'd be glad to add it.